### PR TITLE
Updated "chainhooks service with bitcoind" How To doc

### PR DESCRIPTION
**Pull request by mefrem**
### Description

The ["Run Chainhook as a Service using Bitcoind"](https://docs.hiro.so/chainhook/how-to-guides/how-to-run-chainhook-as-a-service-using-bitcoind) guide was revamped in advance of Chainhooks general launch.

#### Breaking change?

N/A

### Example

N/A

### Checklist

- [ ] All tests pass
- [ ] Tests added in this PR (if applicable)

### CODE REVIEW

File: docs/how-to-guides/how-to-run-chainhook-as-a-service-using-bitcoind.md [hunk 1]
```diff
@@ -8,12 +8,17 @@ You can run Chainhook as a service to evaluate Bitcoin against your predicates.
...
 
-Install bitcoind using [this](https://bitcoin.org/en/bitcoin-core/) link. Ingesting blocks using bitcoind happens through zeromq, an embedded networking library in the bitcoind installation package.
+Bitcoind is a program that implements the Bitcoin protocol for remote procedure call (RPC) use. Chainhook can be set up to interact with the Bitcoin chainstate through bitcoind's ZeroMQ interface, its embedded networking library.
 
-Bitcoind installation will download binaries in a zip format, `bitcoin-22.0-osx64.tar.gz`. You can extract the zip file to view the folders. Expand the `bin` folder to see the bitcoind executable files.
+This guide is written to work with the latest Bitcoin Core software containing bitcoind, [Bitcoin Core 25.0](https://bitcoincore.org/bin/bitcoin-core-25.0/).
+
+> **_NOTE:_**
+>
+> While bitcoind can and will start syncing a Bitcoin node, customizing this node to your use cases beyond supporting a Chainhook is out of scope for this guide. See the Bitcoin wiki for [bitcoind](https://en.bitcoin.it/wiki/Bitcoind), ["Running Bitcoin"](https://en.bitcoin.it/wiki/Running_Bitcoin), or bitcoin.org's [Running A Full Node guide](https://bitcoin.org/en/full-node).
```
Comment (LakshmiLavanyaKasturi) [inline docs/how-to-guides/how-to-run-chainhook-as-a-service-using-bitcoind.md:17]: Did you mean? https://bitcoin.org/ for bitcoin.org's?

File: docs/how-to-guides/how-to-run-chainhook-as-a-service-using-bitcoind.md [hunk 2]
```diff
@@ -106,224 +113,89 @@ tsv_file_url = "https://archive.hiro.so/mainnet/stacks-blockchain-api/mainnet-st
...
-    ```
-    [http_api]
-    http_port = 20456
-    database_uri = "redis://localhost:6379/"
-    ```
-  - Start the Chainhook service by running ```chainhook service start --config-path=Chainhook.toml```.
-  - Now, the predicate registration server is running at `localhost:20456`. To dynamically register a new predicate, send a POST request to `localhost:20456/v1/chainhooks` with the new predicate, in JSON format, included in the request body. For complete documentation on the API endpoints available, see the [OpenAPI](https://raw.githubusercontent.com/hirosystems/chainhook/develop/docs/chainhook-openapi.json) specification.
-  - ![Example post request](../images/chainhook-post-request.jpeg)
-
-The sample payload response should look like this:
```
Comment (LakshmiLavanyaKasturi) [inline docs/how-to-guides/how-to-run-chainhook-as-a-service-using-bitcoind.md:261]: Is it intended to remove the sample payload response?

File: docs/how-to-guides/how-to-run-chainhook-as-a-service-using-bitcoind.md [hunk 3]
```diff
@@ -106,224 +113,89 @@ tsv_file_url = "https://archive.hiro.so/mainnet/stacks-blockchain-api/mainnet-st
...
-
-In this section, you'll learn how to initiate the chainhook service using the following two ways and use the REST API call to post the events onto a server.
-
-- Initiate the chainhook service by passing the predicate path to the command as shown below.
-
-  ```console
-  chainhook service start --predicate-path=ordinals_protocol.json --config-path=Chainhook.toml
-  ```
-
-  The above command registers the predicates based on the predicate definition in the `ordinals_protocol.json` file.
```
Comment (LakshmiLavanyaKasturi) [inline docs/how-to-guides/how-to-run-chainhook-as-a-service-using-bitcoind.md:248]: I think it is important to specify. May I know why we are deleting this?

File: docs/how-to-guides/how-to-run-chainhook-as-a-service-using-bitcoind.md [hunk 4]
```diff
@@ -106,224 +113,89 @@ tsv_file_url = "https://archive.hiro.so/mainnet/stacks-blockchain-api/mainnet-st
...
-
-The above command posts events to the URL, `http://localhost:3000/events` mentioned in the JSON file.
-
-## Initiate Chainhook Service
-
-In this section, you'll learn how to initiate the chainhook service using the following two ways and use the REST API call to post the events onto a server.
-
-- Initiate the chainhook service by passing the predicate path to the command as shown below.
-
-  ```console
```
Comment (LakshmiLavanyaKasturi) [inline docs/how-to-guides/how-to-run-chainhook-as-a-service-using-bitcoind.md:244]: How do we start the service if this is removed?

**Review by LakshmiLavanyaKasturi — CHANGES_REQUESTED** [review]

**Comment by ryanwaits** [comment]
so there's a couple things here that ill create some issues for, but wanted to add here to provide some context since its docs related:

ingesting historical data and triggering actions based on predicates when using Chainhook as a Service:
> there are outstanding questions regarding the behavior of the Chainhook service when dealing with historical data and triggering actions based on registered or scanned predicates

> examples provided in the documentation, including those mentioned here, have not been successful in scanning based on `start_block` and/or `end_block` criteria.

better examples:

> currently, for demonstration purposes, a stacking pool's Bitcoin address is being used to guarantee some data generation. However, Bitcoin related examples are preferred and updates will be made once we resolve current issues around `ordinals_protocol` scope.

File: docs/how-to-guides/how-to-run-chainhook-as-a-service-using-bitcoind.md [hunk 5]
```diff
@@ -2,18 +2,22 @@
 title: Run Chainhook as a Service using Bitcoind
 ---
 
-You can run Chainhook as a service to evaluate Bitcoin against your predicates. You can also dynamically register new predicates by enabling predicates registration API.
+You can run Chainhook as a service to evaluate your `if_this / then_that` predicates against the Bitcoin blockchain, delivering results—either file appendations or HTTP POST requests to a server you designate—for your application's use case. You can also dynamically register new predicates as the service is running by enabling the predicates registration API.
```
Comment (LakshmiLavanyaKasturi) [inline docs/how-to-guides/how-to-run-chainhook-as-a-service-using-bitcoind.md:5]: Chainhook can be used as a service to assess your 'if_this / then_that' predicates against the Bitcoin blockchain. This delivers outcomes by appending files or making HTTP POST requests to your chosen server for your application's needs. You can also dynamically register new predicates as the service runs by enabling the predicates registration API.

**Review by LakshmiLavanyaKasturi — APPROVED** [review]

**Comment by LakshmiLavanyaKasturi** [comment]
@mefrem @ryanwaits: LGTM, just a few re-wordings that I suggested.

**Pull request merged**